alacritty/src/macos/proc.rs

```diff
@@ -1,41 +1,84 @@
-use std::ffi::{CStr, CString};
+use std::ffi::{CStr, CString, IntoStringError};
+use std::fmt::{self, Display, Formatter};
 use std::io;
 use std::mem::{self, MaybeUninit};
-use std::os::raw;
+use std::os::raw::{c_int, c_void};
 use std::path::PathBuf;
 
-pub fn cwd(pid: raw::c_int) -> io::Result<PathBuf> {
+/// Error occuring during getting cwd from `proc_pidinfo`.
```

Comment:
Our errors shouldn't leak implementation details like the API we're using to get the result we want. In theory since this is technically `proc::Error`, this shouldn't even be specific to the `cwd` function. But I'm willing to make that concession for now.

```suggestion
/// Error during working directory retrieval.
```